tests/functional/object/mcg/test_noobaa_db_backup_recovery.py

```
@@ -42,6 +42,20 @@ def determine_snapshot_class_value(self):
             else constants.DEFAULT_VOLUMESNAPSHOTCLASS_RBD
         )
 
+    @pytest.fixture()
```

Ruff (0.15.14)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)